### PR TITLE
chore: remove `VALUE_IS_UNDEFINED` from OData `servicePath` docs

### PR DESCRIPTION
**Pull request by deekshas8**
## What Has Changed?

Update servicePath docs after removing VALUE_IS_UNDEFINED

## Manual Checks?

- [ ] Check spelling and grammar, e.g., using Grammarly.
- [ ] Verify links still work, e.g., if `id` or the name of a file is changed.
- [ ] Update the [feature matrix](https://github.com/SAP/cloud-sdk/blob/main/docs/components/data/features.js), e.g., if a feature is added.

### CODE REVIEW

File: docs-js/features/odata/generate-odata-client.mdx [hunk 1]
```diff
@@ -54,18 +54,19 @@ This will generate OData clients for all your service specifications.
...
 
 - `directoryName`: the name of the subdirectory the client code will be generated into.
-- `servicePath`: the URL path to be prepended before every request.
-  By default, this is read from the EDMX file, if available.
-  Otherwise, the value will be set as `VALUE_IS_UNDEFINED`, and an error will be logged.
-  In this case, you can specify `servicePath` in the `options-per-service.json` file manually and re-generate again.
 - `npmPackageName`: the name of the npm package, if a `package.json` file is generated.
   This information is optional.
+- `servicePath`: the URL path to be prepended before every request.
+  The url is read from the either the EDMX or Swagger file, if available.
```
Comment (FrankEssenberger) [inline docs-js/features/odata/generate-odata-client.mdx:66]:
```suggestion
- `servicePath`: Property to configure the  URL path manually if the automatic determination fails or does not give the desired value.
  The path is prepended before every request.
  If this option is not given the SDK tries to determine the value from either the EDMX or Swagger file.
```

Comment (FrankEssenberger) [reply docs-js/features/odata/generate-odata-client.mdx:66]: Also not super happy with my suggestion but I would like to strengthen the point that in most cases the automatic should be fine.

**Review by FrankEssenberger — APPROVED** [review]
LGTM - consider my suggestion.

**Comment by fwilhe** [comment]
Hi @deekshas8,

we have [user feedback](https://github.com/SAP/cloud-sdk-js/issues/3506#issuecomment-1420549823) on the docs for value is undefined. Can you maybe check with the reporter of this issue if the new docs you're working on is better from their perspective?

Thanks

**Review by FrankEssenberger — APPROVED** [review]

**Pull request merged**